samples/ingress/host-based-ingress/user.env

```diff
@@ -0,0 +1,9 @@
+# This sample demonstrates how to configure 
```

Comment:
These seem a bit excessive to show how to set this single variable but I could be wrong. I think it would be helpful to create a user.env file that shows all the variables and the defaults so that it is easy to see what is configurable similar to how the helm configuration files are done but we can discuss that at a later time.

Reply:
Agreed.  The TLS and the Ingress samples need to be re-worked given our shift to enabling TLS by default.  I'm working on that and will (probably) submit changes in a separate PR.